service/service_setup.go

```diff
@@ -34,21 +34,31 @@ import (
 	"go.uber.org/zap"
 )
 
-func (svc *serviceCore) setupLogging() {
-	err := svc.configProvider.Get("logging").PopulateStruct(&svc.logConfig)
-	if err != nil {
-		zap.L().Info("Logging configuration is not provided, using UberFX defaults", zap.Error(err))
+func (svc *serviceCore) setupLogging() error {
+	cfg := svc.configProvider.Get("logging")
+	if cfg.HasValue() {
+		// populate struct if config was provided
+		zapCfg := zap.Config{Level: zap.NewAtomicLevel()}
```

Comment:
I think it might be better to start with `zap.NewDevelopmentLogger()` and then `PopulateStruct`, so we get all the zap defaults from the library directly rather than copying them here.